app/player/player.service.js

```diff
@@ -516,12 +516,86 @@ module.exports = (db) => {
         }));
     };
 
+    const getActivePlayers = async () => {
+        let players = await db.any(`
+            SELECT * from athlete
+            WHERE active = true
+        `);
+
+        return players.map(p => ({
+            id: p.id,
+            teamId: p.team_id,
+            name: p.name,
+            firstName: p.first_name,
+            lastName: p.last_name,
+            weight: p.weight,
+            height: p.height,
+            jersey: p.jersey,
+            hometownId: p.hometown_id,
```

Comment:
Replace this with the actual hometown fields. Prefer to not return the id from this table.